src/cosalib/ova.py

```diff
@@ -99,9 +99,17 @@ def generate_ovf_parameters(self, vmdk, cpu=2, memory=4096):
         vendor = self.meta['name']
         version = self.meta['ostree-version']
 
+        # Turn off secure boot for RHCOS only
+        # See https://bugzilla.redhat.com/show_bug.cgi?id=2106055
+        if vendor == "rhcos":
```

Comment:
I think this should be a knob in `image.yaml` - that's what it's designed for.  It means we can change this easily without changing coreos-assembler; for example if we want to go back to enabling secure boot in 4.13 or whatever we can do so via a PR to openshift/os, and not break the ability to use coreos-assembler:latest to build older releases.

I am also not excited by introducing *another* "fcos vs rhcos" distinction overall but I see the practical problem here.

Reply:
I'll check out image.yaml